### PR TITLE
A little optimization, increase speed of the client on about 25-30%

### PR DESCRIPTION
**Pull request by cheprasov**

### CODE REVIEW

File: lib/Client.php [hunk 1]
```diff
@@ -187,7 +187,7 @@ public function memory($key, $memory = null, $sampleRate = 1)
             $memory = memory_get_peak_usage();
         }
 
-        $this->count($key, (int) $memory, $sampleRate);
+        $this->count($key, $memory, $sampleRate);
```
Comment (cheprasov) [inline lib/Client.php]: Because, we do convert to `int` in method `$this->count` too.

File: lib/Client.php [hunk 2]
```diff
@@ -241,24 +241,23 @@ public function set($key, $value)
...
-        $message = sprintf("%s:%s|%s", $key, $value, $type);
-        $sample = mt_rand() / mt_getrandmax();
-
-        if ($sample > $sampleRate) {
-            return;
+        if (strlen($this->namespace) !== 0) {
+            $key = $this->namespace . '.' . $key;
         }
 
+        $message = $key . ':' . $value . '|' . $type;
```
Comment (cheprasov) [inline lib/Client.php]: `sprintf` is slower than `string concatenation (with single quotes)` by 163.72% (1.6 times slower)
http://micro-optimization.com/string-concatenation-with-single-quotes-vs-sprintf.html

File: lib/Connection/InetSocket.php [hunk 3]
```diff
@@ -6,6 +6,8 @@
 
 abstract class InetSocket implements Connection
 {
+    const EOL = "\n";
```
Comment (cheprasov) [inline lib/Connection/InetSocket.php]: PHP interpret it only once

**Comment by cheprasov** [comment]
cc @domnikl 
Lets do it :)

**Comment by domnikl** [comment]
Merged, thanks for your contribution!

File: lib/Connection/InetSocket.php [hunk 4]
```diff
@@ -125,13 +123,13 @@ public function send($message)
      */
     public function sendMessages(array $messages)
     {
-        $message = join(self::EOL, $messages);
+        $message = join("\n", $messages);
```
Comment (xjewer) [inline lib/Connection/InetSocket.php]: weird optimization https://3v4l.org/vo5Qb

Comment (cheprasov) [inline lib/Connection/InetSocket.php]: Please, see PR as whole (not just this commit), I checked it and decided not to touch it  :)

Comment (xjewer) [inline lib/Connection/InetSocket.php]: ohh, it so strange, i looked [last commit](https://github.com/domnikl/statsd-php/pull/39/commits/d9a3cf63f9d6a1650c531b7099b5ac0fd5757caf) in pr and it had this changes, but [whole pr](https://github.com/domnikl/statsd-php/pull/39/files) don't affect this lines, wth? )

**Pull request merged**